docs/tutorial/force_calibration.rst

```diff
@@ -104,3 +118,89 @@ We can plot the calibration by calling::
+An approximate expression for the frequency dependent drag coefficient is then given by :cite:`tolic2006calibration`:
+
+.. math::
+
+    \gamma(f, R/l) = \frac{\gamma_{stokes}(f)}{1 - \frac{9}{16}\frac{R}{l}\left(1 - \left((1 - i)/3\right)\sqrt{\frac{f}{f_{\nu}}} + \frac{2}{9}\frac{f}{f_{\nu}}i - \frac{4}{3}(1 - e^{-(1-i)(2l-R)/\delta})\right)}
+
+where :math:`\delta = R \sqrt{\frac{f_{\nu}}{f}}` and :math:`R` corresponds to the bead radius, :math:`l` to the distance from the bead center to the nearest surface.
+
+While these models may look daunting, they are all available in Pylake and can be used by simply providing a few additional arguments to the :class:`~.PassiveCalibrationModel`.
+It is recommended to use these equations when less than 10% systematic error is desired :cite:`tolic2006calibration`.
+
+The figure below shows the difference between the hydrodynamically correct spectrum and the regular Lorentzian for various bead sizes.
+
+.. image:: hydro.png
+
+These more advanced models require a few extra parameters namely the density of the sample, density of the bead and distance to the surface (in meters)::
+
+    force_model = lk.PassiveCalibrationModel(bead_diameter, hydrodynamically_correct=True, rho_sample=999, rho_bead=1060.0, distance_to_surface=1e-6)
+
+Note that when `rho_sample` and `rho_bead` are omitted, values for water and polystyrene are used for the sample and bead density respectively.
+
+Additionally, when the parameter `distance_to_surface` is omitted, a simpler model is used which assumes the experiment was performed deep in bulk (neglecting the increased drag induced by the nearby surface).
+
+Fast Sensors
+------------
+
+Fast detectors provide a higher bandwidth, meaning that they do not need the effect of the diode in the model.
+
+We can omit the diode response model by passing `fast_sensor=True` to the `CalibrationModel`::
+
+    force_model = lk.PassiveCalibrationModel(bead_diameter, viscosity=0.001002, temperature=20, fast_sensor=True, hydrodynamically_correct=True)
+
+Note however, that this makes using the hydrodynamically correct model critical, as the Lorentzian doesn't actually capture the data very well.
+This is illustrated in the figure below.
+Here we see the same power spectrum (acquired on a fast detector) fitted with three different models.
+
+.. image:: fast_hydro.png
+
+Here we can see that the fit with the Lorentzian model with the diode filtering effect seems to fit the data quite well.
+However, when we enable the `fast_sensor` flag, we see that the Lorentzian model doesn't actually describe the data.
+With the diode model disabled, we obtain a very biased fit.
+Enabling the hydrodynamic corrections, we can see that this describes the power spectrum much better.
+If we compare the different fits, we can see that the Lorentzian model with diode effect (fast_sensor=False) gives similar stiffness estimates as the hydrodynamically correct model without the diode effect.
```

Comment:
Could you explain in 1 sentence why the model with the Lorentzian model with diode effect actually gives a good fit?

Reply:
Is two okay? 😅 

```
As we can see in the comparisons above with the hydrodynamics on and off, including hydrodynamics attenuates higher frequencies (an effect similar to a low pass filter).
In the case of the Lorentzian with a diode model, the fitting procedure has used the diode model to fit some of this high frequency attenuation.
```